docs/tech_primer.md

```diff
@@ -0,0 +1,139 @@
+[**UDMI**](../) / [**Docs**](./) / [Tech Primer](#)
+
+# Smart-Ready Building Technical Primer
+
+This document provides  primer for smart-ready building assembly, to essentially
+answer the question *"What does Smart Ready mean?"* and *"How do we know it to be
+true?"* Specifically, it addresses the delta between a traditional site and
+smart-ready sites: places where new bleeding edge building technology is being
+used.
+
+# Key Terms & Definitions
+
+## Building Models
+
+There are different levels of "digital maturity" for different buildings. Each
+one has a slightly different scope in terms of what it means to be "complete."
+For the most part, this document addresses the scope of a Smart Ready building:
+-   **Legacy**: It works, but does not meet requirements or guidelines.
+-   **Compliant**: Meets basic security and networking requirements.
+-   **Smart Ready**: On-prem integrated with the cloud using UDMI & DBO.
+-   **Digital Building**: Completely integrated with back-end services.
+
+## Devices Classification
+
+Various adjectives qualify the different flavors of a "device" present in the
+system. More than one can apply, and each has a specific technical
+meaning/definition (i.e. they are not subjective).
+
+### Connectivity
+
+How data is moved around on-prem
+
+-   **Serial**: The device does not have an IP address, rather uses a wired
```

Comment:
might want to add "wired serial connection to an IoT gateway"

Reply:
sentence added